src/services/directoryServices/ResourceRoomDirectoryService.js

```diff
@@ -51,7 +51,7 @@ class ResourceRoomDirectoryService {
       sessionData
     )
     // If resource room already exists, throw error
-    if ("resources_name" in configContent)
+    if ("resources_name" in configContent && configContent.resources_name)
```

Comment:
is this to account for the case where `configContent.resources_name === ""`?

Reply:
yup it is! the scamshield repo was unable to create a resource room because of this